_includes/js/users.md

```diff
@@ -251,53 +251,53 @@ Using our Facebook integration, you can associate an authenticated Facebook user
-4.  Replace your call to `FB.init()` with a call to `Parse.FacebookUtils.init()`. For example, if you load the Facebook JavaScript SDK asynchronously, your `fbAsyncInit` function will look like this:
+1.  [Create a Facebook Developer account](https://developers.facebook.com/).
+2.  [Create an app](https://developers.facebook.com/apps).
+3.  In your app Dashboard, add a product -> Facebook Login.
+4.  [Add appIds to Parse Server auth configuration](http://docs.parseplatform.org/parse-server/guide/#oauth-and-3rd-party-authentication) or pass `facebookAppIds` into configuration
 
-```javascript
-&lt;script&gt;
+```html
+<script>
   // Initialize Parse
   Parse.initialize("$PARSE_APPLICATION_ID", "$PARSE_JAVASCRIPT_KEY");
+  Parse.serverURL = 'http://YOUR_PARSE_SERVER:1337/parse';
 
-      window.fbAsyncInit = function() {
-    Parse.FacebookUtils.init({ // this line replaces FB.init({
+  window.fbAsyncInit = function() {
+    Parse.FacebookUtils.init({
       appId      : '{facebook-app-id}', // Facebook App ID
       status     : true,  // check Facebook Login status
       cookie     : true,  // enable cookies to allow Parse to access the session
       xfbml      : true,  // initialize Facebook social plugins on the page
       version    : 'v2.3' // point to the latest Facebook Graph API version
     });
-
-        // Run code after the Facebook SDK is loaded.
+    // Run code after the Facebook SDK is loaded.
+    // ...
   };
 
-      (function(d, s, id){
+  // Load Facebook SDK
+  (function(d, s, id){
     var js, fjs = d.getElementsByTagName(s)[0];
     if (d.getElementById(id)) {return;}
     js = d.createElement(s); js.id = id;
     js.src = "//connect.facebook.net/en_US/sdk.js";
     fjs.parentNode.insertBefore(js, fjs);
   }(document, 'script', 'facebook-jssdk'));
-&lt;/script&gt;
+</script>
 ```
 
 The function assigned to `fbAsyncInit` is run as soon as the Facebook JavaScript SDK has completed loading. Any code that you want to run after the Facebook JavaScript SDK is loaded should be placed within this function and after the call to `Parse.FacebookUtils.init()`.
 
 If you encounter any issues that are Facebook-related, a good resource is the [official getting started guide from Facebook](https://developers.facebook.com/docs/reference/javascript/).
 
-If you encounter issues that look like they're being returned from Parse's servers, try removing your Facebook application's App Secret from your app's settings page.
-
-There are two main ways to use Facebook with your Parse users: (1) logging in as a Facebook user and creating a `Parse.User`, or (2) linking Facebook to an existing `Parse.User`.
+There are two main ways to use Facebook with your Parse users: (1) [logging in as a Facebook user](#login--signup) and creating a `Parse.User`, or (2) [linking Facebook](#linking) to an existing `Parse.User`.
```

Comment:
A `ParseUser` can be associated with Facebook either at signup or by linking an existing user.

* [Signing up with Facebook](#login--signup)
* [Linking an existing user to Facebook](#linking)

Reply:
maybe?